cg/meta/archive/ddn/ddn_data_flow_client.py

```diff
@@ -114,13 +114,16 @@ def archive_files(self, files_and_samples: list[FileAndSample]) -> int:
             url=urljoin(base=self.url, url=DataflowEndpoints.ARCHIVE_FILES),
         ).job_id
 
-    def retrieve_samples(self, samples_and_destinations: list[SampleAndDestination]) -> int:
+    def retrieve_files(self, files_and_samples: list[FileAndSample]) -> int:
```

Comment:
A little confused reading this why you would want to retrieve files given that you already have the files and samples but probably the `FileAndSample` model encapsulates meta data

Reply:
The reason is that we have the database model for the file but not the actual file on Hasta. A bit uncertain how to make it more readable, though.